tests/ci/submit_azureml_pytest.py

```diff
@@ -71,10 +71,10 @@ def setup_workspace(workspace_name, subscription_id, resource_group, cli_auth,
     Returns:
         ws: workspace reference
     """
-    logger.debug('setup: workspace_name is ', workspace_name)
-    logger.debug('setup: resource_group is ', resource_group)
-    logger.debug('setup: subid is ', subscription_id)
-    logger.debug('setup: location is ', location)
+    logger.debug('setup: workspace_name is {0}'.format(workspace_name))
```

Comment:
I suggest leveraging logging's [internal formatting](https://docs.python.org/3/library/logging.html#logging.Logger.debug) as it looks cleaner. e.g.:
```
logger.debug('setup: resource group is %s', resource_group)
```

Reply:
agree with improved look. changed to aml norm of:
logger.debug('setup: bla'.format(arg))

thanks